packages/core/src/driver.ts

```diff
@@ -156,7 +161,8 @@ export namespace Driver {
 export abstract class Driver {
   abstract start(): Promise<void>
   abstract stop(): Promise<void>
-  abstract drop(): Promise<void>
+  abstract drop(table: string): Promise<void>
+  abstract dropAll(): Promise<void>
```

Comment:
To avoid breaking change, you can overload this method:

- `drop()` -> dropAll
- `drop(table)` -> dropTable